docs/source/plugin-development/index.rst

```diff
@@ -30,7 +30,8 @@ To get started writing a |Flake8| :term:`plugin` you first need:
 
 Once you've gathered these things, you can get started.
 
-All plugins for |Flake8| must be registered via `entry points`_. In this
+All plugins for |Flake8| must be registered via
+:external:doc:`entry points<specifications/entry-points>`. In this
```

Comment:
Please user the `:external+<name>:doc:...` syntax for this, e.g, `:external+setuptools:`